mscore/inspector/inspector_instrchange.ui

```diff
@@ -54,12 +57,18 @@
       </font>
      </property>
      <property name="text">
-      <string>Change Instrument</string>
+      <string>Instrument Change</string>
```

Comment:
I'd prefer not to change this, because the element is "Change Instrument".

Reply:
Actually, no. Check the status bar. It's the same way that the rehearsal mark inspector is not called "B1."

Reply:
Alright. But I see that in 3.3.4 the text is already changed to "Instrument Change"??

Reply:
That's because currently this file is entirely unused in favor of the Staff Text inspector with the title "Instrument Change."